dlog/logger.go

```diff
@@ -47,13 +50,38 @@ type Logger interface {
 	StdLogger(LogLevel) *log.Logger
 
 	// Log actually logs a message.
-	Log(level LogLevel, args ...interface{})
+	Log(level LogLevel, msg string)
```

Comment:
Reverting this will break Telepresence, which I assume is OK, given that we're not using a released version.